### PR TITLE
Fix fetch * return wrong result when the vertex with empty props tag

### LINKED ISSUE #2351: no data is returned for `fetch prop on *`
**Describe the bug (__must be provided__)**

No data is returned for `fetch prop on * 1`. But `fetch prop on <tagname> <vid>` has the expected result. 

**Your Environments (__must be provided__)**

Docker-compose (with the nightly image) + Studio v1.1.1 beta

**How To Reproduce(__must be provided__)**

I have two data files:  
* user.csv: contains one column only. int, and from 0 to 7000.
* course.csv: contains two columns. one is of the int type (range: from 0 to 96), and the other is of the string type.

1. run `create tag user()` to create a tag without a property.
2. import data from user.csv.
3. run `fetch prop on * 1`, and the expected result is returned.
4. run `create tag course(id int, name string)` to create another tag, named course. Hash("string") is used to generate VIDs.
5. import data from course.csv. 
6. run `fetch prop on * 1`. no data is returned. But when I run `fetch prop on user 1`, the expected result is returned. 

**Expected behavior**

After both user and course data is imported, when I run `fetch prop on * 1`, I should get the expected user data. 

**Additional context**

NO

### PR DESCRIPTION
**Pull request by laura-ding**
close #2351

### CODE REVIEW

File: src/graph/FetchVerticesExecutor.cpp [hunk 1]
```diff
@@ -118,9 +118,6 @@ Status FetchVerticesExecutor::prepareTags() {
             if (!tagIdStatus.ok()) {
                 return tagIdStatus.status();
             }
```
Comment (CPWstatic) [inline src/graph/FetchVerticesExecutor.cpp:120]: Seems we don't need it anymore.

Comment (laura-ding) [reply src/graph/FetchVerticesExecutor.cpp:120]: when `fetch prop on * vid yield tag.prop` need to check the yield

Comment (CPWstatic) [reply src/graph/FetchVerticesExecutor.cpp:120]: I mean the tag id is no more used here.

Comment (laura-ding) [reply src/graph/FetchVerticesExecutor.cpp:120]: Good catch!

**Review by CPWstatic — APPROVED** [review]

**Review by Shylock-Hg — APPROVED** [review]

**Pull request merged**